content/hardware/02.hero/boards/zero/features.md

```diff
@@ -8,23 +8,18 @@ bootloader programming.
 
 <FeatureList>
 
-<Feature title="Cortex-M0 32-bit SAMD21" image="core">
+<Feature title="Embedded Debugger" image="mcu">
 
-The powerful, low-power processor from Microchip.
-
-  <FeatureLink title="Datasheet" url="https://content.arduino.cc/assets/mkr-microchip_samd21_family_full_datasheet-ds40001882d.pdf"/>
-</Feature>
-
-<Feature title="Embedded Debugger (EDBG)" image="mcu">
-
-The Zero comes with a **built-in debugger**, which can be used directly with the Arduino IDE 2.0 version to debug programs.
+The Zero comes with a built-in debugger, which can be used with the Arduino IDE 2.0 version.
 
+  <FeatureLink variant="primary" title="Documentation" url="/tutorials/zero/debugging-with-zero"/>
 </Feature>
 
 <Feature title="Battery Connector" image="power">
 
 The Arduino Zero features a barrel plug connector, that works great with a standard 9V battery.
 
+  <FeatureLink variant="secondary" title="Library" url="https://www.arduino.cc/reference/en/libraries/arduino-low-power/"/>
```

Comment:
I would not link "low power" features in the "battery connector" section. They are related but one is a hardware connector and the other is a library for optimizing the power consumption of the board ;) 

Can you instead add an additional feature, call it low power, and point it to that library?

You can use the following:
- **Title:** Low Power Support
- **Description:** This board can be optimized to reduce power consumption via the Arduino Low Power library